site/_data/deprecation-calendar/deprecation-articles.yml

```diff
@@ -0,0 +1,2 @@
+- deprecation_id: 6239658726391808
+  article: "/articles/declarative-shadow-dom/"
```

Comment:
As it's a 1:1 relationship, consider using the id as a key, this will allow you to simply lookup articles by id further down, instead of needing to do `articles.filter` everytime you want to look one up.